core/jvm/src/test/scala/scalaz/zio/FiberSpec.scala

```diff
@@ -0,0 +1,18 @@
+package scalaz.zio
+
+class FiberSpec(implicit ee: org.specs2.concurrent.ExecutionEnv) extends TestRuntime {
+  def is =
+    "FiberSpec".title ^ s2"""
+    Create a new Fiber and
+      lift it into Managed $e1
+    """
+
+  def e1 = unsafeRun(
+    for {
+      ref   <- Ref.make(false)
+      fiber <- IO.never.ensuring(ref.set(true)).fork
```

Comment:
The test is failing because `never.ensuring` probably doesn't register the finalizer. You could rewrite it as `IO.unit.bracket(ref.set(true))(_ => IO.never).fork`

Reply:
Done. It was blinking probably because of the finalizer, as I noticed it passed for 2.12 and fail for 2.13 after initially committed. Locally, I couldn't reproduce it.